### PR TITLE
added how to use link in README

### PR DESCRIPTION
**Pull request by LordGameleo**
Solves #22

### CODE REVIEW

**Comment by LordGameleo** [comment]
@Razzeee Please review

File: README.md [hunk 1]
```diff
@@ -3,3 +3,9 @@ Your friendly intro and outro detection robot
 
 ## notrobro-detector
 This is the part you can run on your server/against your files to generate a file that defines where intros and outros are
+
+
+## How to use
+[Documentation](https://github.com/xbmc/notrobro/tree/master/notrobro-detector#notrobro-detector)
```
Comment (Razzeee) [inline README.md]: That's just for the detector part. We should make that clear or give instructions for the other part.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,9 @@ Your friendly intro and outro detection robot
...
+
+## This Project is still in developement so there is a lot of scope for you to help us.
```
Comment (Razzeee) [inline README.md]: The `a lot of scope` doesn't seem to be correct english here. Go for something like `Feel free to send us your PR or issues.`

File: README.md (hunk 1 above)
Comment (LordGameleo) [reply README.md]: Other part like!?

Comment (Razzeee) [reply README.md]: The service for kodi itself https://github.com/xbmc/notrobro/tree/master/service.notrobro

Comment (mohit-0212) [reply README.md]: Include that this separately comprises of both the detector (notrobro-detector) and the service for Kodi (service.notrobro) with a bit of info about both and then add their respective links.

**Comment by LordGameleo** [comment]
@Razzeee @mohit-0212 Please give it a look.

**Comment by LordGameleo** [comment]
Is there any changes required? @Razzeee @mohit-0212

File: README.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # notrobro
-Your friendly intro and outro detection robot
+Your friendly intro and outro detection robot. This repository comprises of both the detector (notrobro-detector) and the service for Kodi (service.notrobro).
 
 ## notrobro-detector
-This is the part you can run on your server/against your files to generate a file that defines where intros and outros are
+You can run it on your server/against your files to generate a file that defines where intros and outros are.
```
Comment (Razzeee) [inline README.md]: Not `a file` but `files`

File: README.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
+- [Documentation for detector](https://github.com/xbmc/notrobro/tree/master/notrobro-detector#notrobro-detector)
+
+## service.notrobro
+You can create what you would like Kodi to do! You can create things such as plugins.
+- [Guideline for using service for Kodi](https://github.com/xbmc/notrobro/tree/master/service.notrobro)
```
[the remark this answers is not in the document]
Comment (LordGameleo) [reply README.md]: @mohit-0212 I didn't get you. What all stuffs should be filtered out?

[the remark this answers is not in the document]
Comment (mohit-0212) [reply README.md]: The readme present at [https://github.com/xbmc/notrobro/tree/master/service.notrobro](url) is the default readme generated from the addon-generator tool [https://github.com/xbmc/generator-kodi-addon](url). 
The readme is created the same in general for addon developers from the tool and so doesn't contain -  (1) information pertaining to 'this' service (notrobro), (2) simple how-to-use for users.

**Comment by Razzeee** [comment]
Feel free to rebase and reopen, I'll close this for now.

**Pull request closed without merging**